packages/core/strapi/lib/commands/transfer/import.js

```diff
@@ -42,7 +42,12 @@ module.exports = async (opts) => {
   /**
    * To local Strapi instance
    */
-  const strapiInstance = await strapi(await strapi.compile()).load();
+  let strapiInstance;
+  try {
+    strapiInstance = await strapi(await strapi.compile()).load();
+  } catch (_err) {
+    throw new Error('Process failed. Check the database connection with your Strapi project.');
```

Comment:
Can we check that err is actually a database connection? I assume strapi.compile().load() has a lot of things that could possibly go wrong.